etc/DependencyInstaller.sh

```diff
@@ -69,6 +69,16 @@ _installCommonDev() {
         echo "Eigen already installed."
     fi
 
+    # CUSP
+    if [[ ! -d /usr/local/include/cusp/ ]]; then
```

Comment:
What about the cuda dependencies?

Reply:
I'd advocate that we don't try to install CUDA it's super difficult to lock down the right version for a particular device.

You also run a high risk of screwing something up on the user's machine.

Reply:
Maybe provide a link to where the users should go get the right CUDA downloads?

Reply:
Ok but at least a message would be good as people think the dependencies will all be installed.

Reply:
Is it okay to mention the message here?
![image](https://user-images.githubusercontent.com/57721574/184547602-bbb17bd5-7b43-4294-a720-c382e1b2f518.png)